plugins/libplugin.c

```diff
@@ -1607,7 +1607,11 @@ static void ld_command_handle(struct plugin *plugin,
 		for (size_t i = 0; i < plugin->num_notif_subs; i++) {
 			if (streq(cmd->methodname,
 				  plugin->notif_subs[i].name)
-			    || streq(plugin->notif_subs[i].name, "*")) {
+			    /* * is a special case that means all, but log is another special
+			     * case that can impact performance so this notification is turned
+			     * only when directly specified by a plugin. */
+			    || (streq(plugin->notif_subs[i].name, "*")
+				&& !streq(cmd->methodname, "log"))) {
```

Comment:
Any chance of unifying this and the check above? Now that they are not as trivial anymore, we might want to encapsulate them in a function.

Reply:
yeah we can have somethings in `common`

Reply:
addressed in https://github.com/ElementsProject/lightning/pull/6990/commits/07f8f69f1b565d6f96c08650d86733fc52682476